lib/util/findNextPunctuators.js

```diff
@@ -43,51 +44,31 @@ module.exports = function(node, source, lines) {
     return punctuators;
   }
 
-  function getNextExpressions(source, node) {
+  function getNextExpression(source, node) {
     function findNextExpressionToken(source, node) {
       var isVarAssignment = isVariableAssignment(node), amount = !isVarAssignment ? 2 : 1;
       return source.getTokenAfter(!isVarAssignment ? node.parent : node, amount);
     }
-    function findProperPunctuator(next, x) {
-      var after = source.getTokenAfter(next, 2 - x);
-      if (after.value !== '=') {
-        return findProperPunctuator(after, 1);
-      }
-      return after;
-    }
 
-    var result = [], punctuators = getPunctuators(node, source), next = findNextExpressionToken(source, node);
+    var next = findNextExpressionToken(source, node);
     if (typeof next === 'undefined') {
       if (isVariableAssignment(node) && node.declarations.length > 1) {
-        return punctuators;
+        return [];
```

Comment:
this is nnot needed anymore. THe ``punctuators`` will be computed in ``getDeclarations`` and everything will end up in the ``finally`` clause, so this condition should be removed, but the error should be thrown at every time when ``next`` is ``undefined``